tests/common.go

```diff
@@ -684,6 +685,45 @@ func ValidatePDSDataServices(ctx *scheduler.Context, errChan ...*chan error) {
 	})
 }
 
+func IsPoolAddDiskSupported() (bool, error) {
+
+	DMthin, err := IsDMthin()
+	if err != nil {
+		return true, nil
```

Comment:
How about?

```go
dmthin, err := IsDMthin()
if err != nil {
	return false, err
}
if !dmthin {
	return true, nil
} else {
	// Rest of the implementation
}
```

Reply:
This is not required because if IsDmThin returns false that means its btrfs and we support it .

Reply:
If IsDMthin returns an error, it would be assumed that it failed to determine if the backend is dmthin, not that the backend is btrfs.

Reply:
added , Thanks..